sa/type-converter.go

```diff
@@ -38,6 +38,16 @@ func (tc BoulderTypeConverter) ToDb(val interface{}) (interface{}, error) {
 		return string(t), nil
 	case core.OCSPStatus:
 		return string(t), nil
+	case *core.AcmeURL:
```

Comment:
Agreed JWS should be a blob, I think it still needs a type converter though, same with `*core.AcmeURL`, as inserting and scanning were both broken for me without converters.